src/component/AutoComplete.tsx

```
@@ -0,0 +1,133 @@
+        <span className="text-muted-foreground typography-body-r">
+          {text.substring(index + query.length)}
+        </span>
+      </>
+    );
+  };
+
+  return (
+    <div className="relative" ref={containerRef}>
+      <label className="block text-sm font-medium text-foreground mb-2">
+        {label}
+      </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)